packages/database-mongo/package.json

```diff
@@ -32,38 +22,24 @@
   "files": ["src", "lib"],
   "jest": {
     "transform": {
-      ".(ts|tsx)": "<rootDir>/node_modules/ts-jest/preprocessor.js"
+      ".(ts|tsx)": "<rootDir>/../../node_modules/ts-jest/preprocessor.js"
     },
     "testRegex": "./__tests__/.*.ts$",
     "moduleFileExtensions": ["ts", "js"]
   },
   "repository": {
     "type": "git",
-    "url": "https://github.com/js-accounts/mongo.git"
+    "url": "https://github.com/accounts-js/accounts.git"
   },
   "author": "Leo Pradel",
   "license": "MIT",
   "bugs": {
-    "url": "https://github.com/js-accounts/mongo/issues"
+    "url": "https://github.com/accounts-js/accounts/issues"
   },
-  "homepage": "https://github.com/js-accounts/mongo",
+  "homepage": "https://github.com/accounts-js/accounts",
   "devDependencies": {
     "@accounts/common": "0.1.0-beta.3",
-    "@accounts/server": "0.1.0-beta.3",
-    "@accounts/tslint-config-accounts": "0.0.6",
-    "@types/jest": "22.2.0",
-    "@types/lodash": "4.14.104",
-    "@types/mongodb": "3.0.7",
```

Comment:
you removed some needed types

Reply:
I red that if a package exports it's types, it is not needed to include his typing package in devDeps

Reply:
is mongo exporting his types directly?

Reply:
lodash is not exporting it's types

Reply:
Seems to not be... I don't understand why tsc didn't crash then ...

Reply:
it's because we are not using strict mode, will be great to have but will need some work before

Reply:
I see, actually I am struggling with this, those typings mess up the compile function...

Reply:
Is it needed to also revert @accounts/tslint-config-accounts and @types/jest ?

Reply:
In that case no :)